lime-core/src/main/scala/org/bdgenomics/lime/serialization/LimeKryoRegistrator.scala

```diff
@@ -9,8 +9,22 @@ class LimeKryoRegistrator extends KryoRegistrator {
   private val akr = new ADAMKryoRegistrator()
 
   override def registerClasses(kryo: Kryo) {
+    kryo.register(classOf[scala.Array[org.bdgenomics.adam.models.ReferenceRegion]])
```

Comment:
Unfortunately, right now we are having to individually register each class, and there is a problem using ADAM's kryo serialization.